templates/content/taxonomy-term--site-themes.html.twig

```diff
@@ -25,10 +25,11 @@
 #}
 <div{{ attributes.setAttribute('role', 'presentation') }}>
   {{ drupal_block('page_title_block', wrapper=false) }}
+  {{ content | without('field_additional_info') }}
   {% if information_services_output is defined %}
     {{ information_services_output }}
-  {%  else %}
-    {{ content }}
   {%  endif %}
-  {{ content['field_additional_info'] }}
+  {% if content['field_additional_info'] %}
+    {{ content['field_additional_info'] }}
```

Comment:
Is the if needed here, if there is no additional field it wouldn't render anything?

Reply:
Too long with my head in the PHP cloud :) 

I vaguely recall Twig being more graceful when handling empty values. I'll try it and if it works, I'll drop the surrounding if statement.